packages/core/src/dropdown/Dropdown.css

```diff
@@ -23,15 +28,28 @@
   font-weight: var(--salt-text-fontWeight);
   line-height: var(--salt-text-lineHeight);
   letter-spacing: var(--salt-text-letterSpacing);
+  position: relative;
+  overflow: hidden;
 }
 
 .saltDropdown:hover {
   --dropdown-borderColor: var(--salt-editable-borderColor-hover);
+  --dropdown-borderStyle: var(--salt-editable-borderStyle-hover);
+  --dropdown-background: var(--saltDropdown-background-hover, var(--dropdown-background-hover));
+}
+
+.saltDropdown:active {
+  --dropdown-background: var(--saltDropdown-background-active, var(--dropdown-background-active));
+  --dropdown-borderColor: var(--salt-editable-borderColor-active);
+  --dropdown-borderWidth: var(--salt-editable-borderWidth-active);
+  --dropdown-borderStyle: var(--salt-editable-borderStyle-active);
 }
 
-.saltDropdown:focus-visible {
```

Comment:
This change also needs to be documented. The focus ring is shown on focus rather than focus-visible

Reply:
added to changeset